submissions/Nemooochka/html-css-popup/css/style.css

```
@@ -0,0 +1,138 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #fefefe;
+    color: #8a8a8a;
+    font: 16px/1.2 Montserrat, sans-serif;
+}
+
+a {
+    color: #8a8a8a;
+    text-decoration: none;
+}
+
+input[type="checkbox"]:focus ~ label {
+    outline: 1px solid #005fcc;
+}
+
+.container {
+    max-width: 1120px;
+    margin: 0 auto;
+    padding: 0 15px;
+}
+
+label {
+    cursor: pointer;
+}
+
+header nav {
+    display: flex;
+    justify-content: flex-end;
+}
+
+.nav-menu {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    list-style: none;
+}
+
+.nav-menu li {
+    position: relative;
+    padding: 0 12px;
+    font-size: 18px;
+}
+
+.nav-menu li a,
+.nav-menu li .toggleMenu_label {
+    padding: 0 4px;
+}
+
+.nav-menu .with-icon {
+    font-size: 1.2em;
+}
+
+.toggleInput {
+    position: absolute;
+    left: -9999px;
+}
+
+.nav-menu li #toggleMenu:checked ~ label {
+    color: #ef7c7c;
+}
+
+.nav-menu li #toggleMenu:checked ~ .popup-menu {
+    display: block;
+}
+
+.popup-menu {
+    display: none;
+    position: absolute;
+    width: 400px;
+    max-height: 380px;
+    top: 40px;
+    left: 50%;
+    transform: translateX(-50%);
+    background-color: #fff;
+    border: 1px solid #8a8a8a;
+    border-radius: 3px;
+    z-index: 999;
+    overflow-y: scroll;
+}
+
+.menu {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    padding: 20px;
+    list-style: none;
+}
+
+.menu li {
+    width: 33.333%;
+    margin: 5px 0;
+}
+
+.menu li a {
```

You can assign some class name for this link and simplify this selector
```suggestion
.someClassName {
```